resolver-dns/src/main/java/io/netty/resolver/dns/NoopDnsCache.java

```diff
@@ -46,16 +47,17 @@ public boolean clear(String hostname) {
     }
 
     @Override
-    public List<DnsCacheEntry> get(String hostname) {
+    public List<DnsCacheEntry> get(String hostname, Iterable<DnsRecord> additional) {
         return Collections.emptyList();
     }
 
     @Override
-    public void cache(String hostname, InetAddress address, long originalTtl, EventLoop loop) {
+    public void cache(String hostname, Iterable<DnsRecord> additional,
+                      InetAddress address, long originalTtl, EventLoop loop) {
     }
 
     @Override
-    public void cache(String hostname, Throwable cause, EventLoop loop) {
+    public void cache(String hostname, Iterable<DnsRecord> additional, Throwable cause, EventLoop loop) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1186)